eth/vm/computation.py

```diff
@@ -149,9 +140,31 @@ def is_origin_computation(self) -> bool:
         """
         return self.msg.sender == self.transaction_context.origin
 
+    #
+    # Program Counter
+    #
+    def get_pc(self) -> int:
+        return max(0, self.code.pc - 1)
+
     #
     # Error handling
     #
+    @property
+    def error(self) -> Optional[VMError]:
```

Comment:
Maybe this `error` refactor could be split out

Reply:
cc @rayrapetyan though I will look into doing this myself tomorrow and you can rebase your branch.